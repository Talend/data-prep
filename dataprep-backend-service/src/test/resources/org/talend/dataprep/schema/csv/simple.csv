first name;last name
James;Butt
Josephine;Darakjy
Art;Venere
Lenna;Paprocki
Donette;Foller
Simona;Morasca
Mitsue;Tollner