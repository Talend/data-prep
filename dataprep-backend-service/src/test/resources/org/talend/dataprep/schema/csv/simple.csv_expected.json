[
  {
    "0000": "James",
    "0001": "Butt"
  },
  {
    "0000": "Josephine",
    "0001": "Darakjy"
  },
  {
    "0000": "Art",
    "0001": "Venere"
  },
  {
    "0000": "Lenna",
    "0001": "Paprocki"
  },
  {
    "0000": "Donette",
    "0001": "Foller"
  },
  {
    "0000": "Simona",
    "0001": "Morasca"
  },
  {
    "0000": "Mitsue",
    "0001": "Tollner"
  }
]